src/index.html

```diff
@@ -11,7 +11,166 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form method="get"
+    action="https://mate-academy-form-lesson.herokuapp.com/create-application">
+      <fieldset>
+        <legend>
+          Personal information
+        </legend>
+        <div>
+          <label>
+            Surname:
+            <input
+            type="name"
+            autocomplete="off"
+            name="usersurname"
+            >
```

Comment:
```suggestion
            <input
              type="name"
              autocomplete="off"
              name="userName"
            >
```
you didn't fix code style
<img width="923" alt="image" src="https://user-images.githubusercontent.com/102411130/223352344-207450bb-62a8-4eb1-8b32-b7d64c02614c.png">